### PR TITLE
Minor appearance changes to messages and replies.

### PR DESCRIPTION
**Pull request by amuletspade231**
Changed reply buttons to say "Comment" to match placeholder text.
Changed message button to say "Post."
Added indent margins to replies.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -93,6 +93,9 @@ public void storeReply(Message reply) {
     replyEntity.setProperty("recipient", reply.getRecipient());
     replyEntity.setProperty("sentimentScore", reply.getSentimentScore());
     replyEntity.setProperty("timestamp", reply.getTimestamp());
+    if(reply.getImageUrl() != null) {
+      replyEntity.setProperty("imageURL", reply.getImageUrl());
+    }
```
Comment (dj-smith) [inline src/main/java/com/google/codeu/data/Datastore.java]: So replies can have images uploaded? Is there a way for users to actually do that?

File: src/main/webapp/js/user-page-loader.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ function fetchGallery() {
       } else {
         messagesContainer.innerHTML = '';
         messages.forEach((message) => {
-          const messageDiv = buildMessageDiv(message);
+          const messageDiv = buildMessageDiv(message, "0px");
```
Comment (dj-smith) [inline src/main/webapp/js/user-page-loader.js]: Thanks for catching this case

File: src/main/webapp/js/user-page-loader.js [hunk 3]
```diff
@@ -184,7 +184,7 @@ function fetchAboutMe() {
  * @param {Message} message
  * @return {Element}
  */
-function buildMessageDiv(message) {
+function buildMessageDiv(message, margin) {
```
Comment (dj-smith) [inline src/main/webapp/js/user-page-loader.js]: Should probably update the comment of this to include the new param and/or an explanation of how the function works

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (amuletspade231) [reply src/main/java/com/google/codeu/data/Datastore.java]: Hmm I'll have to look at Gloria's code, but I could probably make it happen!

File: src/main/webapp/js/user-page-loader.js (hunk 3 above)
Comment (amuletspade231) [reply src/main/webapp/js/user-page-loader.js]: Will do!

**Pull request merged**